frontend/package.json

```diff
@@ -11,6 +11,7 @@
     "@types/react": "^18.3.5",
     "@types/react-dom": "^18.3.0",
     "bootstrap": "^5.3.3",
+    "moment": "^2.30.1",
```

Comment:
Did you add this on purpose? what does it do?

Reply:
We added it to parse the date, but ended up doing this another way, we will remove it